packages/turf-meta/index.js

```diff
@@ -822,6 +822,128 @@ function feature(geometry, properties) {
     };
 }
 
+
+/**
+ * Callback for segmentEach
+ *
+ * @callback segmentEachCallback
+ * @param {Feature<LineString>} [currentSegment] The current segment being processed.
+ * @param {number} [currentIndex] The index of the current element being processed in the array, starts at index 0.
+ * @returns {void}
+ */
+
+/**
+ * Iterate over 2-vertex line segment in any GeoJSON object, similar to Array.forEach()
+ *
+ * @param {FeatureCollection|Feature|Geometry} geojson any GeoJSON
+ * @param {Function} callback a method that takes (currentSegment, currentIndex)
+ * @returns {void}
+ * @example
+ * var polygon = {
```

Comment:
👍 Agreed, I really do prefer these "shorter" examples, more focused on the code itself and not 10 lines of defining a GeoJSON polygon.